src/GameSrc/setup.c

```diff
@@ -26,7 +26,6 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define __SETUP_SRC
 
 #include <string.h>
-#include <unistd.h>
```

Comment:
Needed for my build. What I'd do is probably add a wrapping header that includes <unistd.h> on POSIX platforms but does whatever it takes to get access() on Windows. Also preferably defining an F_OK macro while it's at it.

Reply:
I'll leave a suggested compatibility precompiler block here but leave it up to you to put it wherever it should go.